From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Changki Kim <changki.kim@samsung.com>
Date: Thu, 2 Jul 2020 10:20:49 +0900
Subject: NOUPSTREAM: ANDROID: PM: sleep: export device_pm_callback_start/end
 trace event

Export the device_pm_callback_start/end tracepoint so it can be used
in loadable modules.

[CPNOTE: 17/06/21] Lee: Vendor related code - maintain forever

Bug: 158528747

Change-Id: Icfe12f496f25d3b73dbd70394dc85d197709671a
Signed-off-by: Changki Kim <changki.kim@samsung.com>
(cherry picked from commit d16111a8cf537426853300149b2ce65790fb650c)
Signed-off-by: Jonglin Lee <jonglin@google.com>
---
 kernel/trace/power-traces.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/kernel/trace/power-traces.c b/kernel/trace/power-traces.c
--- a/kernel/trace/power-traces.c
+++ b/kernel/trace/power-traces.c
@@ -18,4 +18,5 @@ EXPORT_TRACEPOINT_SYMBOL_GPL(suspend_resume);
 EXPORT_TRACEPOINT_SYMBOL_GPL(cpu_idle);
 EXPORT_TRACEPOINT_SYMBOL_GPL(cpu_frequency);
 EXPORT_TRACEPOINT_SYMBOL_GPL(powernv_throttle);
-
+EXPORT_TRACEPOINT_SYMBOL_GPL(device_pm_callback_start);
+EXPORT_TRACEPOINT_SYMBOL_GPL(device_pm_callback_end);
